scripts/foundry/Deploy.s.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity >=0.8.0 <0.9.0;
+pragma solidity >=0.8.24 <0.9.0;
```

Comment:
why this change only at one place

Reply:
I just replaced all 0.8.0 by 0.8.24

Reply:
I thought it was ^0.8.24 already for all files

you mean all files have
pragma solidity >=0.8.24 <0.9.0; now

Reply:
this was just replace accross all files, but this one had different pragma I'll revert it

Reply:
got it